xm-commons-logging-web/src/main/java/com/icthh/xm/commons/logging/web/aop/RestLoggingAspect.java

```diff
@@ -175,12 +211,39 @@ private void logStop(final JoinPoint joinPoint, String method, final String[] co
             return;
         }
 
-        log.info("STOP  {} : {} --> {}, result: {}, time = {} ms",
-                 method,
-                 LogObjectPrinter.joinUrlPaths(controllerPath, methodPath),
-                 LogObjectPrinter.getCallMethod(joinPoint),
-                 WebLogObjectPrinter.printRestResult(joinPoint, result),
-                 MdcUtils.getExecTimeMs());
+        className = joinPoint.getSignature().getDeclaringType().getSimpleName();
```

Comment:
in every place extract joinPoint.getSignature() and joinPoint.getSignature().getDeclaringType() to variable

Reply:
done